src/pybind/mgr/cephadm/migrations.py

```diff
@@ -181,6 +181,11 @@ def migrate_2_3(self) -> bool:
         return True
 
     def migrate_nfs_spec(self, service_id: str, pool: str, ns: Optional[str]) -> None:
+        renamed = False
+        if service_id.startswith('ganesha-'):
+            service_id = service_id[8:]
+            renamed = True
+
```

Comment:
We have asked users to create a new cluster. Then is the migration still required?
https://github.com/ceph/ceph/blob/master/PendingReleaseNotes#L111-L114

Reply:
updated release notes